pkgs/development/python-modules/pytest-benchmark/default.nix

```diff
@@ -70,18 +80,24 @@ buildPythonPackage rec {
     export PATH="$out/bin:$PATH"
   '';
 
+  pytestFlagsArray = [
+    "-W"
+    "ignore::DeprecationWarning"
+  ];
+
   disabledTests = [
+    # https://github.com/ionelmc/pytest-benchmark/issues/231
+
+    # Failed: nomatch: '*collected 5 items'
+    "test_abort_broken"
+    # TypeError
+    "test_clonefunc"
+  ] ++ lib.optionals (pythonOlder "3.12") [
```

Comment:
So they fail on 3.8, 3.9 and 3.10 as well? Or should this be `pythonAtLeast "3.11"`?